### ISSUE TITLE
Drag to pan while zoomed in

### ISSUE THREAD

**Issue by tst5381** [neo4j-labs/arrows.app#7]
Usually I zoom in to work on a specific area and then need to work on another, currently I must zoom out and then zoom in, which is quite tedious. The functionality to pan without zoom in/out or changing the scale would be much appreciated.

**Comment by jexp** [comment]
yes ctrl-drag would be nice for that.

**Comment by apcj** [comment]
Hi @tst5381 , I'm a bit confused by this. Are you not able to pan using mouse scroll wheel? How are you zooming?

**Comment by tst5381** [comment]
@apcj With my mouse wheel I can only scroll vertically. I also tried pressing the wheel button to switch the cursor-scrolling mode on, but pointing to horizontally direction doesn't pan the canvas.

**Comment by apcj** [comment]
Right, I see what you mean @tst5381 . We had wrongly assumed that most people have mice with horizontal scrolling these days. Will think about how to handle this. Would you consider using a trackpad?

**Comment by guntarsTB** [comment]
The same problem for me too.
I have a mouse with a tilting scroll wheel, but it just doesn't work really. It allows a horizontal scroll for just a small pan movement, i.e. I would have to tilt the scroll wheel 50 times (holding doesn't work - must tilt it and release each time) to pan the diagram significantly. There desperately needs to be another simple way to pan it.
It could be a right-click (like in Miro app) or it could be CTRL/ALT/SHIFT whatever combination with a left click, or it could be a middle-click. Keyboard would also do. Just ANY method to pan the diagram. Otherwise I have to zoom out and zoom in, and it works in an unpredictable manner. Disabling the "auto zoom" would also be convenient, otherwise the diagram keeps panning and zooming itself and it is disturbing.

**Comment by CrisBMoya** [comment]
How is this open and unattended after 5 years?
